components/blitz/src/omero/cmd/mail/SendEmailRequestI.java

```diff
@@ -85,6 +85,10 @@ public void init(Helper helper) {
         rsp.invalidusers = new ArrayList<Long>();
         rsp.invalidemails = new ArrayList<String>();
 
+        if (!everyone && groupIds.size() < 1 && userIds.size() < 1)
```

Comment:
`groupIds.isEmpty()`, etc. is more idiomatic (or, if it can be null, we have `CollectionUtils.isEmpty` which is null-safe)